chatminer/chatparsers.py

```diff
@@ -261,17 +262,17 @@ def _read_raw_messages_from_file(self):
                 self._logger.info(
                     'No chat name was specified, searching for chat "Saved Messages"...'
                 )
+                # ? Does this take long enough to add tqdm around `json_objects`
```

Comment:
I don't think so. It iterates over a small list and does not do any logic besides a condition.

Reply:
I will remove this in my next commit.